examples/conform.py

```diff
@@ -59,7 +59,8 @@ def parse_args():
         type=str,
         required=True,
         help='Timeline file(s) to read. Any format supported by OTIO will'
-        ' work.'
+        ' work: fcp_xml, fcpx_xml, EDL, AAF, and more via community '
```

Comment:
You can take this a step further if you'd like and list the available adapters, using this function:
https://github.com/PixarAnimationStudios/OpenTimelineIO/blob/9d010f2966523505b5c954d803f4f982f5e4edb5/src/py-opentimelineio/opentimelineio/console/otioconvert.py#L36

And if you want to put the file suffixes in the message, you can use this function:
https://github.com/PixarAnimationStudios/OpenTimelineIO/blob/9d010f2966523505b5c954d803f4f982f5e4edb5/src/py-opentimelineio/opentimelineio/adapters/__init__.py#L52

Its a great idea to put that information into the file input argument!  That upgrade could go into the real scripts (otioconvert, otioview, etc.) if you were interested in making the fix.

Reply:
On it! Thank you for the info!